charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -80,6 +85,9 @@ spec:
               -sds-server-host {{ template "consul.fullname" . }}-api-gateway-controller.{{ .Release.Namespace }}.svc \
               -k8s-namespace {{ .Release.Namespace }} \
               -log-level {{ default .Values.global.logLevel .Values.apiGateway.logLevel }} \
+              {{- if (and .Values.global.metrics.enabled .Values.global.metrics.enableGatewayMetrics) }}
```

Comment:
Not sure if `global.metrics.enableGatewayMetrics` makes sense here since we're dealing with **_controller_** metrics and not _**gateway**_ metrics per se.

Reply:
This is the main thing I'm looking for feedback on here. My understanding shifted after initially talking to @Jeff-Apple , but it sounded like we still wanted this in ahead of GA

Reply:
Yeah, so I'm ok exposing the metrics if we so desire, but my thought is that we could just key off of `global.metrics.enabled` rather than both it and `global.metrics.enableGatewayMetrics` since we're talking about controller-level metrics. I would think that as far as deployed api gateways are concerned we'd key off of both, but for the controller, at least IMO (which isn't super informed here 😅 ) the one makes sense, but not necessarily the other.

I can totally envision someone wanting to scrape controller metrics but not all deployed gateways, and tying the two options together this way would preclude that possibility without adding yet another flag/values option down the line.